web/modules/custom/customhook/customhook.api.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+// This is a documentation of the customhook created for ticket FT2023-340
+// The purpose of this hook is to display the number of times a node is viewed
+// over a session
```

Comment:
We should not be using double slashes here.
Please compare with some other api.php file in core/.